code/datums/martial/krav_maga.dm

```diff
@@ -86,7 +86,7 @@
 	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
 				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.losebreath += 5
+	D.losebreath = Clamp(D.losebreath + 5, 0, 10)
```

Comment:
Clamp returns max if the first argument is greater than max.
If losebreath was 11 before the punch, Clamp(D.losebreath + 5, 0, 10) will return 10 because losebreath + 5 >= 10.